test/support/ssl/insecure-cert.pem

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----
+MIIF9DCCA9ygAwIBAgIJAJhLP5JrvONkMA0GCSqGSIb3DQEBBQUAMFkxCzAJBgNV
```

Comment:
that's not really what I was asking... if we fail to specify a cert (as in one doesn't exist and we don't create one) will `Security.certificateError` be triggered. I lack the time to investigate or I would test that, but it should be verified before we proceed. If we don't need a cert in the repo to test that, then we shouldn't include one.

If we must provide a failing cert, then we should utilize https://github.com/davewasmer/devcert or equivalent to generate a cert at the time of testing (and cleanup afterwards) rather than including one in the repo statically. webpack-dev-server just went through a security mess with that.